### PR TITLE
New mods

### PR DESCRIPTION
**Pull request by sakshamAK**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,13 +23,16 @@ function App() {
         <Route path="mockbee" element={<Mockman />} />
         <Route path="/" element={<Home />} />
         <Route path="/store" element={<ProductList />} />
```
Comment (himadri2110) [inline src/App.js:25]: Suggestion: You can extract the `Routes` component into a separate file, to make the `App.js` clean.

**Review by himadri2110 — APPROVED** [review]
The carousel on the Home page looks cool.
Good to go!

File: src/App.js (hunk 1 above)
Comment (SJTGSHIVAM) [reply src/App.js:25]: This is debatable, in a project of this size I believe breaking it into a new file will reduce readability. Yes, it is a good move if the project grows. I would suggest deferring this decision.

**Review by SJTGSHIVAM — APPROVED** [review]
Code is functionally correct and good to merge.

**Pull request merged**